src/main/java/io/github/syst3ms/skriptparser/Parser.java

```diff
@@ -153,7 +153,7 @@ public static void printLogs(List<LogEntry> logs, Calendar time, boolean tipsEna
         for (LogEntry log : logs) {
             ConsoleColors color = ConsoleColors.WHITE;
             if (log.getType() == LogType.WARNING) {
-                color = ConsoleColors.YELLOW;
+                color = ConsoleColors.ORANGE;
```

Comment:
i dont think change to orange is a good idea since yellow is in general associate with warning so it would imo create bad idea of warning for skript users that would next  switch to another language